backend/app/Models/Account.php

```diff
@@ -71,4 +72,18 @@ public function getBalance()
 
         return 0;
     }
+
+    public function accountExist(string $accountNumber, string $accountName)
+    {
+        $account = $this->where('account_number', $accountNumber)->first();
+
+        if ($account) {
+            similar_text(Str::lower($account['account_name']), Str::lower($accountName), $percentSimilarity);
```

Comment:
use similar_text method from php, apply 80% similarity to fuzzy match the  input